draft-ietf-scitt-architecture.md

```diff
@@ -192,6 +192,11 @@ Auditor:
 
 : an entity that checks the correctness and consistency of all Transparent Statements issued by a Transparency Service.
 
+Client:
+
+: a process, user or another service that externally connects to the Transparency Service.
+A Client and Issuer are not synonymous as a Client may register Statements signed by a different Issuer identity.
```

Comment:
Is a Verifier a Client? If not, does a Verifier use a Client? I bring this up because in my prior work on this I presented a updated diagram to confirm my understanding and I was corrected on this point. However, this seems an important place to clarify this and/or in the definition of Verifier, otherwise the clarification is not in the current draft of the document.

Reply:
I would simply write that the Issuer may or may not be the client that submits the signed claim to a TS for registration. (The proposed text suggests the client may also sign, which seems irrelevant here.)

Reply:
If you read the original report in #102, your recommendation (from my perspective, others can feel free to disagree) further confuses the purpose of the original request for clarification: what a client (what is doing it) is unclear. Casually blurring the line between what is doing and who is doing (issuer, verifier has been unclear to me if it what or who) is part of the problem here.

Reply:
With my proposed definition above I would just add 

> (who may or may not be the Issuer of the statement)

at the end of my proposed definition above

Reply:
I still have concerns given the explanations to answer and fully resolve 102, but my questions have been answered and they are outside the scope of my support of the PR recommendation and tentative changes proposed in https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/pull/197/files#r1488069804.